src/Fabulous.MauiControls.Tests/WidgetTests.fs

```diff
@@ -56,7 +56,8 @@ type WidgetTests() =
               GetViewNode = ViewNode.get
               Logger = ProgramDefaults.defaultLogger()
               Dispatch = dispatch
-              GetComponent = Component.get }
+              GetComponent = Component.get
+              SyncAction = fun fn -> fn() }
```

Comment:
You can call MainThread here because it is a Maui test project

Reply:
Nice. Thanks